makefile

```diff
@@ -1,6 +1,22 @@
 MODULE=weevenetwork/serial-data-decryptor
 VERSION_NAME=v1.0.0
 
+create_image:
+	docker build -t ${MODULE}:${VERSION_NAME} . -f docker/Dockerfile
+.phony: create_image
+
+run_image:
+	docker run -p 80:80 --rm --env-file=./.env ${MODULE}:${VERSION_NAME}
+.phony: run_image
+
+debug_image:
+	docker run -p 80:80 --rm --env-file=./.env --entrypoint /bin/bash -it ${MODULE}:${VERSION_NAME}
+.phony: debug_image
+
+push_latest:
+	docker image push ${MODULE}:${VERSION_NAME}
+.phony: push_latest
```

Comment:
pushing single image for current arch?

Reply:
this will probably never be used. I'll remove it